example/pubspec.yaml

```diff
@@ -3,15 +3,19 @@ description: Demonstrates how to use the audio_session plugin.
 
 # The following line prevents the package from being accidentally published to
 # pub.dev using `pub publish`. This is preferred for private packages.
-publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+publish_to: "none" # Remove this line if you wish to publish to pub.dev
 
 environment:
   sdk: ">=2.17.0 <4.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  just_audio: ^0.9.31
+  just_audio:
+    git:
+      url: git@github.com:arturograu/just_audio.git
```

Comment:
This was the only way to make the example compile since `just_audio` depends on `audio_session`. So `audio_session` cannot be upgraded to `rxdart` 0.28.0 until `just_audio` upgrades.

I suggest creating a pre-release with the content of this PR so we can upgrade `just_audio` at least depending on this pre-release. Then we can change this dependency back into the official `just_audio` and do the official release.